statSTR/README.md

```diff
@@ -1 +1,31 @@
-** Under construction **
\ No newline at end of file
+# StatSTR 
+
+This tool is used for computing stats on VCF files 
+
+## Usage 
+To run statSTR use the following command: 
+```
+statSTR 
+  --vcf <vcf file> \
+  --out test.tab \
+  --thresh \
+  [filter options]
```

Comment:
I would say "additional options" rather and "filter options" since here we are computing statistics rather than actually filtering